### PR TITLE
[JS] Remove support for Edge HTML

### PR DESCRIPTION
**Pull request by harsha509**
### Description
Removing edge legacy support as discussed in [here](https://github.com/SeleniumHQ/selenium/issues/9166) 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [x] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: javascript/node/selenium-webdriver/chrome.js [hunk 1]
```diff
@@ -284,7 +284,6 @@ function getDefaultService() {
 Options.prototype.CAPABILITY_KEY = 'goog:chromeOptions'
 Options.prototype.BROWSER_NAME_VALUE = Browser.CHROME
 Driver.getDefaultService = getDefaultService
-Driver.prototype.VENDOR_COMMAND_PREFIX = 'goog'
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/chrome.js]: The vendor prefix is still being used on Chromium based browsers like Edge Chromium and Chrome. Did you mean to remove this?

File: javascript/node/selenium-webdriver/edge.js [hunk 2]
```diff
@@ -65,134 +56,121 @@
...
-    return this
+  setEdgeChromiumBinaryPath(path) {
+    return this.setBinaryPath(path)
   }
 }
 
-Options.USE_EDGE_CHROMIUM = 'ms:edgeChromium'
-Options.prototype.BROWSER_NAME_VALUE = Browser.EDGE
-Options.prototype.CAPABILITY_KEY = 'ms:edgeOptions'
-Options.prototype.VENDOR_CAPABILITY_PREFIX = 'ms'
```
Comment (AutomatedTester) [inline javascript/node/selenium-webdriver/edge.js:142]: I think this is used by the bidi tests to know where to find the websocket address for us to connect to.

**Review by AutomatedTester — COMMENTED** [review]
This is looking good and it would be good to see if the tests run the bidi tests in `devtools_test.js`

File: javascript/node/selenium-webdriver/chrome.js (hunk 1 above)
Comment (harsha509) [reply javascript/node/selenium-webdriver/chrome.js]: I have overlooked into it. I have reverted the change. Thank you for review @AutomatedTester !

File: javascript/node/selenium-webdriver/edge.js (hunk 2 above)
Comment (harsha509) [reply javascript/node/selenium-webdriver/edge.js:142]: Reverted it.

**Comment by harsha509** [comment]
Thank you @AutomatedTester for the review. I have reverted the change. 

Also observed the debugger address for edgeChromium is like below:

'ms:edgeOptions' => { debuggerAddress: 'localhost:50588' },

so added `caps['map_'].get(this.VENDOR_CAPABILITY_PREFIX + ':edgeOptions') ||`  in the cdpConnectionCreation. 

Thank you!

**Review by AutomatedTester — APPROVED** [review]

**Pull request merged**